spec/unit/utils_spec.rb

```diff
@@ -29,13 +29,13 @@
     context 'when public' do
       let(:access) { 'world' }
 
-      it { is_expected.to be 'public' }
+      it { is_expected.to eq 'public' }
     end
 
     context 'when restricted' do
       let(:access) { 'stanford' }
 
-      it { is_expected.to be 'restricted' }
+      it { is_expected.to eq 'restricted' }
     end
```

Comment:
These tests were failing locally for me, possible equality of strings vs object problem (presumably we care about the value of the string, and not the object itself being the same).